config/jobs/kubernetes-csi/gen-jobs.sh

```diff
@@ -24,23 +24,23 @@ base="$(dirname $0)"
 # irrelevant because the prow.sh script will pick a suitable KinD
 # image or build from source.
 k8s_versions="
-1.24
-1.25
 1.26
+1.27
+1.28
 "
 
 # All the deployment versions we're testing.
 deployment_versions="
-1.24
-1.25
 1.26
+1.27
+1.28
 "
 
 # The experimental version for which jobs are optional.
-experimental_k8s_version="1.26"
+experimental_k8s_version="1.28"
 
 # The latest stable Kubernetes version for testing alpha jobs
-latest_stable_k8s_version="1.25" # TODO: bump to 1.26 after testing a pull job
+latest_stable_k8s_version="1.26" # TODO: bump to 1.26 after testing a pull job
```

Comment:
We missed updating it for 1.27, so change "bump to 1.26" -> "bump to 1.28".

Reply:
@xing-yang I assume the change here was tested and then changed to 1.28 in another pr? According to your previous reply. Did I misunderstand?
![image](https://github.com/kubernetes/test-infra/assets/9926112/7c26eb41-2a8e-4f23-a69b-8ccf21f25a75)